packages/pwa-buildpack/lib/WebpackTools/plugins/RootComponentsPlugin.js

```diff
@@ -7,12 +7,16 @@ const micromatch = require('micromatch');
 
 const prettyLogger = require('../../util/pretty-logger');
 
-const toRootComponentMapKey = (type, variant = 'default') =>
-    `RootCmp_${type}__${variant}`;
+const toRootComponentMapKey = function(type, variant) {
+    variant = variant || 'default';
+    return 'RootCmp_' + type + '__' + variant;
```

Comment:
I'm surprised the linter didn't complain, but we should avoid mutating arguments. I suggest deleting L11 and revising L12.

```suggestion
    return 'RootCmp_' + type + '__' + (variant || 'default');
```